src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,42 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    user,
+    comments,
+    body,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
+        </p>
+      </div>
+
+      <p className="PostInfo__body">
+        {body}
+      </p>
+
+      <hr />
+
+      {comments.length === 0 ? (
+        <>
+          <hr />
+          <b data-cy="NoCommentsMessage">No comments yet</b>
+        </>
```

Comment:
I think you don't need `<hr />` in this case, it's too much
![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/63243136/a29b0a29-6617-4aec-86f1-f53e3eaae604)

Reply:
fixed👍